src/Middleware/FailureHandling/Implementation/ExponentialDelayMiddleware.php

```diff
@@ -63,7 +63,7 @@ public function processFailure(
         $message = $request->getMessage();
         if ($this->suites($message)) {
             $messageNew = new Message(
-                handlerName: $message->getHandlerName(),
+                handlerName: $message->getHandler(),
```

Comment:
I think it should be also renamed in this case `handler: $message->getHandler(),`